packages/shared/friends/sagas.ts

```diff
@@ -1716,12 +1781,18 @@ export function muteChannel(muteChannel: MuteChannelPayload) {
     store.dispatch(unmutePlayers([channelId]))
   }
 
+  let onlineMembers = 0
+  if (channel.userIds) {
+    const userStatuses = client.getUserStatuses(...channel.userIds)
+    onlineMembers += [...userStatuses.values()].filter((status) => status.presence === PresenceType.ONLINE).length
+  }
+
```

Comment:
This is duplicating the logic in lines 399 and forth. Let's extract an util function for counting online users.

Reply:
It's done in this PR: https://github.com/decentraland/kernel/pull/614/files#diff-bcea93a768b74b3a44000978a9f8e8f2b257395933ac042653b8153f0a1c751fR1979